manifests/profiles/external.yaml

```diff
@@ -0,0 +1,22 @@
+# The external profile is used to configure a mesh using an external control plane.
+# Only the injector mutating webhook configuration is installed.
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  components:
+    base:
+      enabled: false
+    pilot:
+      enabled: false
+    ingressGateways:
+    - name: istio-ingressgateway
+      enabled: false
+    istiodRemote:
+      enabled: true
+  values:
+    telemetry:
+      enabled: false
+    pilot:
+      configMap: false
```

Comment:
I am not sure about this. It depends on who you want to control the mesh config (control plane cluster or user cluster).

Reply:
That's only an option if it's also the config cluster. I was trying to make this profile for a non-config cluster using an external control plane. Then a config-cluster would add the CRDs and optionally the configmap to the profile:
```
# config cluster with an external control plane
apiVersion: install.istio.io/v1alpha1
kind: IstioOperator
spec:
  profile: external
  components:
    base:
      enabled: true # install CRDs for config cluster
    pilot:
      configMap: true # add the istio configmap
```

Maybe, since external control plane config clusters will be more common, it should work the other way around, i.e., they're set by default and the non-config cluster would turn them off.
```
# non-config cluster with an external control plane
apiVersion: install.istio.io/v1alpha1
kind: IstioOperator
spec:
  profile: external
  components:
    base:
      enabled: false # don't install CRDs
    pilot:
      configMap: false # no istio configMap
```

I'd be OK either way.

@howardjohn WDYT?

Reply:
Oh good point. This seems ok to me then, would like to get others feedback